### PR TITLE
VcfHeaderLoaderStepParametersValidator_EVA-606

### PR DESCRIPTION
**Pull request by poggio84**

### CODE REVIEW

File: src/test/java/uk/ac/ebi/eva/pipeline/parameters/validation/InputStudyNameValidatorTest.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    private JobParametersBuilder jobParametersBuilder;
+
+    @Before
+    public void setUp() throws Exception {
+        validator = new InputStudyNameValidator();
+    }
+
+    @Test
+    public void inputStudyNameIsValid() throws JobParametersInvalidException {
```
Comment (cyenyxe) [inline src/test/java/uk/ac/ebi/eva/pipeline/parameters/validation/InputStudyNameValidatorTest.java]: Test multiple strings with different types of characters (digits, symbols, white-spaces, etc.).

File: src/test/java/uk/ac/ebi/eva/pipeline/parameters/validation/InputStudyTypeValidatorTest.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    public void caseStudyTypeIsValid() throws Exception {
+        JobParametersBuilder jobParametersBuilder = new JobParametersBuilder();
+        jobParametersBuilder.addString(JobParametersNames.INPUT_STUDY_TYPE, "CASE");
+        validator.validate(jobParametersBuilder.toJobParameters());
+    }
+
+    @Test
+    public void caseControlStudyTypeIsValid() throws Exception {
+        JobParametersBuilder jobParametersBuilder = new JobParametersBuilder();
+        jobParametersBuilder.addString(JobParametersNames.INPUT_STUDY_TYPE, "CASE_CONTROL");
```
Comment (cyenyxe) [inline src/test/java/uk/ac/ebi/eva/pipeline/parameters/validation/InputStudyTypeValidatorTest.java]: Nothing to change here, but I was just wondering if we should support more user-friendly strings like "case-control" and any possible capitalization of it.

Comment (poggio84) [inline src/test/java/uk/ac/ebi/eva/pipeline/parameters/validation/InputStudyTypeValidatorTest.java]: Ah yes good idea... the same in _org.opencb.biodata.models.variant.VariantSource.Aggregation_

File: src/test/java/uk/ac/ebi/eva/pipeline/parameters/validation/InputStudyNameValidatorTest.java (hunk 1 above)
Comment (poggio84) [inline src/test/java/uk/ac/ebi/eva/pipeline/parameters/validation/InputStudyNameValidatorTest.java]: Done in next commit

**Pull request merged**